chroma_core/models/jobs.py

```diff
@@ -331,6 +331,9 @@ def __hash__(self):
     states = ("pending", "tasked", "complete")
     state = models.CharField(max_length=16, default="pending", help_text="One of %s" % (states,))
 
+    depends_on_job_range = models.CharField(max_length=64, default="[]", help_text="Range of jobs in list that must\
```

Comment:
Why does this need to be saved to the db?

Reply:
Looking at it further I think I can do it without storing the data. I'll update.